storybook/shared/box.js

```diff
@@ -29,12 +29,12 @@ export default function Box({
 }: BoxProps) {
   return (
     <Grid
-      margin={typeof margin === 'undefined' ? [0, 0.5, 1] : margin}
+      margin={typeof margin === 'undefined' ? [0, 'L/2', 'L'] : margin}
```

Comment:
From this point on it's updates to stories to use aliases instead of numerical values